oktapam/resource_ad_task_settings.go

```diff
@@ -181,10 +181,12 @@ func resourceADTaskSettingsRead(ctx context.Context, d *schema.ResourceData, m i
 	var diags diag.Diagnostics
 	c := m.(client.OktaPAMClient)
 
-	id := d.Id()
-	adConnId := d.Get(attributes.ADConnectionID).(string)
+	adConnId, adTaskSettingsId, err := parseADTaskSettingsResourceID(d.Id())
```

Comment:
So this can have a d[conneciton_id], and d.Id() is just ad task settings ID?

Reply:
Done. Fixed in [6c0355a](https://github.com/okta/terraform-provider-oktapam/pull/65/commits/6c0355a9175e980f2658c6d23769aa347eac8d8f)